src/test/java/com/google/javascript/gents/multiTests/goog_provide_rewrite/export.ts

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Can you special-case the newline insertion so that it doesn't create one as the first line in a file?